labcodes/django-react-vite
# Fix/node 16 lts

**How to test:**
- reinstall dependencies (you may use the `make install_dependencies` command);
- run the frontend tests and make sure they run successfully (`npm test`);
- run the app and check if it still works correctly (`make run`);
- build the frontend app and verify the build doesn't return any errors.

**Description of your solution:**

Since there's a new Node LTS version (v16), and it brings a new dependency checker, the js app wouldn't install anymore, because of outdated packages and configs.

This PR:
- updates packages with broken dependencies;
- fixes vulnerabilities;
- reconfigures the js test environment to match new dependencies;
- reconfigures eslint to run on jsx files;
- reconfigures eslint to allow dynamic imports;
- lints js(x) files with new `prettier` rules.
